nemo/collections/common/tokenizers/char_tokenizer.py

```diff
@@ -88,6 +107,9 @@ def ids_to_text(self, ids):
     def tokens_to_ids(self, tokens):
```

Comment:
please fix the add_special_tokens() method to return the number of special tokens added so that we can later adjust the model size properly. Please see nemoGPT2 implementation https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/common/tokenizers/gpt2_tokenizer.py#L57 and HF:https://github.com/huggingface/transformers/blob/77abd1e79fc37efb45bde0041f9dad0eabc55517/src/transformers/tokenization_utils_base.py#L673 and

Reply:
Please sync with @AlexGrinch about that after PR merge